### PR TITLE
Add Postgres Resources section to menu

### PR DESCRIPTION
**Pull request by saltcod**

### CODE REVIEW

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -611,6 +611,58 @@ export const resources = {
...
+          url: '/guides/resources/supabase-cli/local-development',
+          items: [],
+        },
+        {
+          name: 'Managing Environments',
+          url: '/guides/resources/supabase-cli/managing-environments',
+          items: [],
+        },
+      ],
+    },
```
Comment (dannykng) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:633]: @saltcod I assume you didn't mean to add Supabase CLI back to the Resources section.
```suggestion
```

Comment (saltcod) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:633]: aah right! thanks

**Comment by awalias** [comment]
@saltcod I added a new FooterCTACallout, for these postgres pages can we show it instead of the FooterHelpCallout

File: apps/docs/pages/guides/resources/postgres.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'postgres-resources',
+  title: 'Postgres Resources',
+  description:
+    'Resources to help you get up to speed with Postgres, the database powering Supabase.',
+}
+
+postgres index page
```
Comment (MildTomato) [inline apps/docs/pages/guides/resources/postgres.mdx:10]: delete page?

File: apps/docs/pages/guides/resources/postgres/index.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Postgres resources',
+  description:
+    'Resources to help you get up to speed with Postgres, the database powering Supabase.',
+}
+
+Postgres resources
```
Comment (MildTomato) [inline apps/docs/pages/guides/resources/postgres/index.mdx:9]: delete page?

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/docs/pages/guides/resources/postgres/dropping-all-tables-in-schema.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+begin
+    for r in (select tablename from pg_tables where schemaname = 'my-schema-name') loop
+        execute 'drop table if exists ' || quote_ident(r.tablename) || ' cascade';
+    end loop;
+end $$;
+```
+
+This snippet works by first listing out all the tables in the given schema. And then executing a `drop table` for each, hence the `for... loop`.
+
+You can run this query from the [SQL editor](https://app.supabase.com/project/_/sql) inside the Supabase dashboard, or via psql if [connecting directly via the command line](/docs/guides/database/connecting-to-postgres#direct-connections).
```
Comment (dannykng) [inline apps/docs/pages/guides/resources/postgres/dropping-all-tables-in-schema.mdx:31]:
```suggestion
This query works by listing out all the tables in the given schema and then executing a `drop table` for each (hence the `for... loop`).

You can run this query using the [SQL Editor](https://app.supabase.com/project/_/sql) in the Supabase Dashboard, or via `psql` if you're [connecting directly to the database](/docs/guides/database/connecting-to-postgres#direct-connections).
```

File: apps/docs/pages/guides/resources/postgres/first-row-in-group.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+We want to find the rows containing the maximum number of points _per team_.
+
+The expected output we want is:
+
+| id  |   team    | points |
+| --- | :-------: | -----: |
+| 3   | Brighton  |     34 |
+| 2   | Liverpool |     84 |
+
+From the [SQL editor](https://app.supabase.com/project/_/sql) you can run a query like:
```
Comment (dannykng) [inline apps/docs/pages/guides/resources/postgres/first-row-in-group.mdx:28]:
```suggestion
From the [SQL Editor](https://app.supabase.com/project/_/sql), you can run a query like:
```

File: apps/docs/pages/guides/resources/postgres/first-row-in-group.mdx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  seasons
+order BY
+  id,
+  points desc,
+  team;
+```
+
+The important bits here are the `desc` that we put on the ordering of `points` which ranks the points in descending value (starting with the highest). And the `distinct` keyword that tells Postgres to only return a single row per team.
+
+This query can also be executed via psql or any other query editor if you prefer to [connect directly to the database](/docs/guides/database/connecting-to-postgres#direct-connections).
```
Comment (dannykng) [inline apps/docs/pages/guides/resources/postgres/first-row-in-group.mdx:45]:
```suggestion
The important bits here are:
- The `desc` keyword to order the `points` from highest to lowest.
- The `distinct` keyword that tells Postgres to only return a single row per team.

This query can also be executed via `psql` or any other query editor if you prefer to [connect directly to the database](/docs/guides/database/connecting-to-postgres#direct-connections).
```

File: apps/docs/pages/guides/resources/postgres/which-version-of-postgres.mdx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+export const meta = {
+  title: 'Print PostgreSQL version',
+  description: 'Useful snippet for finding out which version of postgres you are running',
+  footerHelpType: 'postgres',
+}
+
+It's important to know which version of PostgreSQL you are running, since each major version has different features and may cause breaking changes. If [upgrading](https://www.postgresql.org/docs/current/pgupgrade.html) or downgrading a major Postgres version, you may need to update your schema.
+
+From the [SQL editor](https://app.supabase.com/project/_/sql) you can run the simple query:
```
Comment (dannykng) [inline apps/docs/pages/guides/resources/postgres/which-version-of-postgres.mdx:11]:
```suggestion
It's important to know which version of PostgreSQL you are running as each major version has different features and may cause breaking changes. You may also need to update your schema when [upgrading](https://www.postgresql.org/docs/current/pgupgrade.html) or downgrading to a major Postgres version.

Run the following query using the [SQL Editor](https://app.supabase.com/project/_/sql) in the Supabase Dashboard:
```

File: apps/docs/pages/guides/resources/postgres/which-version-of-postgres.mdx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  version ();
+```
+
+Which should return something like:
+
+```sql
+PostgreSQL 15.1 on aarch64-unknown-linux-gnu, compiled by gcc (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0, 64-bit
+```
+
+This query can also be executed via psql or any other query editor if you prefer to [connect directly to the database](/docs/guides/database/connecting-to-postgres#direct-connections).
```
Comment (dannykng) [inline apps/docs/pages/guides/resources/postgres/which-version-of-postgres.mdx:24]:
```suggestion
This query can also be executed via `psql` or any other query editor if you prefer to [connect directly to the database](/docs/guides/database/connecting-to-postgres#direct-connections).
```

**Review by dannykng — APPROVED** [review]
Looks good! Just one minor styling issue to fix: https://github.com/supabase/supabase/pull/11926/files#r1093816430

File: apps/docs/components/FooterHelpCallout.tsx [hunk 9]
```diff
@@ -1,11 +1,38 @@
-const FooterHelpCallout = () => {
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown'
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/docs/components/FooterHelpCallout.tsx:1]: on it

[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/docs/components/FooterHelpCallout.tsx:1]: fixed

**Comment by MildTomato** [comment]
@awalias can merge when builds all pass

**Review by awalias — APPROVED** [review]

**Review by MildTomato — APPROVED** [review]

**Pull request merged**